UnityProject/Assets/Scripts/NPC/AI/Hostile/FaceHugAction.cs

```diff
@@ -115,6 +115,7 @@ public static bool HasAntihuggerItem(Equipment equipment)
 					if (!Validations.HasItemTrait(item.gameObject, CommonTraits.Instance.AntiFacehugger))
 					{
 						Inventory.ServerDrop(itemSlot);
+						antiHugger = true; //It just tries again
```

Comment:
Why is this change needed?

Reply:
I would say it's a balancing thing so it's relatively subjective,
 I think part of the issue is that we have such low pop that  the dancing idea of "well the station has loads of crew they can just bum rush the aliens doesn't really work" so this gives you an opportunity if you're skilled you can somewhat survive by yourself
 
 but it subjective so I'm interested to hear your opinion

Reply:
I think our mobs are already really dumb and easily cheesed. If they don't offer a challenge or aren't a threat then they might as well not be there.

This is specially important in very low pop scenarios, because the players need an antagonist and these games normally turn to PvE.

Reply:
Do you remember old xenos? They would chase you down with such an aggression that players actually felt threatened by them. They would break windows and tables if they had eyes on you.

Nowadays people can grab the xeno queen and throw her to disposals like its nothing.

Reply:
yeah,  though that's mainly because we stuck between two systems currently

Reply:
quick interjection
don't fix or work on old mobs, they're deprecated.
nuke them
work on facehuggers and xenos from scratch with a better base that's not slop
#10500